lookout/style/format/analyzer.py

```diff
@@ -43,6 +44,43 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
         :param data: Contains "changes" - the list of changes in the pointed state.
         :return: List of comments.
         """
+        def vnode_to_comment(vnode: VirtualNode, correct_y: int) -> str:
```

Comment:
`vnode_to_error_description`

Even though we document everywhere that we return a string, everybody will still expect this func to return a Comment

Reply:
mm, since we move out it from `VirtualNode` class I think we can shorten and name it just `error_description` WDYT?

Reply:
There must be a verb either at the beginning or at the end